### ISSUE TITLE
CDN configuration from the site general settings is ignored

### ISSUE THREAD

**Issue by kumards** [Lombiq/Combinator#17]
There is a configuration "Use CDN" under site general settings is ignored when combinator is enabled. It always picks the local version and falls back to cdn only when local resource url is not set.
I see the below method from ResourceDefinitionExtensions.cs is used to get the full path which has no access to the site settings.

```
 public static string GetFullPath(this ResourceDefinition resource)
        {
            if (string.IsNullOrEmpty(resource.Url)) return resource.UrlCdn;

            if (resource.Url.Contains("~")) return resource.Url;

            return Path.Combine(resource.BasePath, resource.Url);
        }
```

**Comment by Piedone** [comment]
Yes, that feature was added later than I've implemented this in Combinator so it indeed causes this incompatibility.

However if most of your static resources come from CDNs there is not too much point in bundling them locally.

**Comment by kumards** [comment]
I do have lot of js files that require bundling and minification where I find Combinator very handy. Only the js libraries need CDN support.
